testsuite/integration/microprofile-tck/lra/src/test/resources/arquillian-bootable.xml

```diff
@@ -5,22 +5,22 @@
   -->
 
 <arquillian xmlns="http://jboss.org/schema/arquillian" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+            xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
 
-    <defaultProtocol type="jmx-as7" />
+    <!-- Note: Using 'jmx-as7' instead of 'Servlet' eliminates the need to grant extra permissions to deployments when running with security manager enabled. -->
+    <defaultProtocol type="jmx-as7"/>
```

Comment:
FYI @jamezp Continuing on discussion from Friday regarding deprecating the jmx-as7 protocol, removing this line is the reproducer for the issue I mentioned where the remote command simply fails with a timeout when using the ARQ command dispatching mechanism when running from a Servlet (with no indication anywhere, since ARQ likes (or liked?) to eat up exceptions). Also, all of this will be moot point, once the security manager is removed altogether.